### DISCUSSION TITLE
Tried playing around with the UI, and wondering if libadwaita breaks things

### DISCUSSION THREAD

**Discussion by vancha** [qarmin/czkawka#1233] [Ideas]
I was trying to make some changes to czkawka to see if i could make it a little more in line with the operating system I use. In doing so, i tried changing the main gtk4 application that this app uses, to libadwaita's adw::Application. That seems to work, and on gnome that automatically adds support for dark mode when it's enabled system wide. Is this something that anyone is interested in? 
If so, should i make a pull request? Result can be seen in the attached screenshot, please ignore the missing icons:
![Screenshot from 2024-03-05 16-15-00](https://github.com/qarmin/czkawka/assets/9658702/5dbbae97-5e21-4748-85f1-fb9e6986d6b0)

**Comment by qarmin** [comment]
libadwaita is an additional dynamically linked library that is in very different versions in repositories - https://repology.org/project/libadwaita/versions, so using it could give different results on different systems.
I've had enough bugs with GTK to know to stay away from it.

**Reply by vancha** [reply]
Alright nevermind then, introducing more bugs than it would fix would be the last thing anyone needs haha. Thanks for this awesome app, and keep up the good work ^^
